examples/preprocessing/plot_artifact_detection.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+=============================================
+Detect artifacts
+=============================================
```

Comment:
This will break Circle I think. The number of == should match the number of characters in the title

Reply:
Actually this should be ok, the requirement is that the line of symbols must be *at least as long* as the text.